.eslintignore

```diff
@@ -36,5 +36,4 @@ exercises/secret-handshake
 exercises/simple-cipher
 exercises/simple-linked-list
 exercises/triangle
```

Comment:
Just another thing. Maybe I'm mixing up PRs. Shouldn't this PR remove `triangle` from `.eslintignore`?